mapreduce/src/main/java/org/apache/rya/accumulo/mr/RyaOutputFormat.java

```diff
@@ -216,12 +215,22 @@ private static FreeTextIndexer getFreeTextIndexer(Configuration conf) {
         return freeText;
     }
 
-    private static TemporalIndexer getTemporalIndexer(Configuration conf) {
+    private static TemporalIndexer getTemporalIndexer(Configuration conf) throws IOException {
         if (!conf.getBoolean(ENABLE_TEMPORAL, true)) {
             return null;
         }
         AccumuloTemporalIndexer temporal = new AccumuloTemporalIndexer();
         temporal.setConf(conf);
+        Connector connector;
+        try {
+            connector = ConfigUtils.getConnector(conf);
+        } catch (AccumuloException | AccumuloSecurityException e) {
+            throw new IOException("Error when attempting to create a connection for writing the temporal index.", e);
+        }
+        MultiTableBatchWriter mtbw = connector.createMultiTableBatchWriter(new BatchWriterConfig());
```

Comment:
Why are you creating the multitable batch writer here?  Can't you use the same  batchwriter that the other indexes are using?  Have your getTemporalIndexer(...) method take in a MultiTableBatchWriter as well.